src/engine/guidance/post_processing.cpp

```diff
@@ -608,7 +608,7 @@ std::vector<RouteStep> buildIntersections(std::vector<RouteStep> steps)
     {
         auto &step = steps[step_index];
         const auto instruction = step.maneuver.instruction;
-        if (instruction.type == TurnType::Suppressed)
+        if (instruction.type == TurnType::Suppressed || instruction.type == TurnType::UseLane)
```

Comment:
I've removed the usage completely (except for a single location which exposes it to the scripting environment to not break profiles). At least the tests pass alright, so I would be inclined towards the complete rip-out you suggested.